src/nvme/ioctl.h

```diff
@@ -3848,7 +3848,7 @@ struct nvme_io_args {
 	__u16 apptag;
 	__u16 appmask;
 	__u8 dsm;
-	__u8 dspec;
+	__u16 dspec;
```

Comment:
Can you move the dspec before dsm so that we have all __u16 grouped together. This avoids holes in the data structures.

Reply:
addressed :)

Reply:
You need also to update all inline function in the header which use struct nvme_io_args. Older compilers are complaining if the order of the struct initialization are not strictly following the order of the deceleration.